spec/features/admin/reports_spec.rb

```diff
@@ -204,28 +204,28 @@
 
     it "reports" do
       # Then it should give me access only to managed enterprises
-      page.should     have_select 'q_distributor_id_eq', with_options: [user1.enterprises.first.name]
-      page.should_not have_select 'q_distributor_id_eq', with_options: [user2.enterprises.first.name]
+      expect(page).to     have_select 'q_distributor_id_eq', with_options: [user1.enterprises.first.name]
+      expect(page).not_to have_select 'q_distributor_id_eq', with_options: [user2.enterprises.first.name]
 
       # When I filter to just one distributor
       select user1.enterprises.first.name, from: 'q_distributor_id_eq'
       click_button 'Search'
 
       # Then I should see the relevant order
-      page.should have_content "#{order1.number}"
+      expect(page).to have_content "#{order1.number}"
 
       # And the totals and sales tax should be correct
-      page.should     have_content "1512.99" # items total
-      page.should     have_content "1500.45" # taxable items total
-      page.should     have_content "250.08" # sales tax
-      page.should     have_content "20.0" # enterprise fee tax
+      expect(page).to     have_content "1512.99" # items total
+      expect(page).to     have_content "1500.45" # taxable items total
+      expect(page).to     have_content "250.08" # sales tax
+      expect(page).to     have_content "20.0" # enterprise fee tax
```

Comment:
Now that the tabulation is not aligned with the paragraphs above and below I'd simply replace it with a single space.